Enterprise-CMCS/macpro-mdct-mcr
# Fix bug where creating an MLR report caused an error.

### Description
Not 100% sure how this happened, but someone (most likely me) created a bug in MLR that made it so you didn't get your  full report data/form template back when creating a report, causing the `setReport` function to set an incomplete report object.

This is now fixed.

### Related ticket(s)


N/A

---
### How to test

1. Navigate to MLR
2. Create a report
3. Verify it doesn't error.

### Important updates


`create.ts` now returns full report data on report creation. This is a change, but doesn't seem to invalidate any existing tests.

---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
